docs/jwst/tweakreg/README.rst

```diff
@@ -105,6 +126,21 @@ The ``tweakreg`` step has the following optional arguments:
 * ``sigma``: A positive `float` indicating the clipping limit, in sigma units,
   used when performing fit. (Default=3.0)
 
+**Astrometric fitting parameters:**
+
+* ``align_to_gaia``: A boolean indicating whether or not to fit to
+  perform the fit to an astrometric catalog. (Default=False)
+
+* ``gaia_catalog``: String indicating what astrometric catalog should be used.
+  Initially supported options:{'GAIADR2', 'GAIADR1'}. (Default='GAIADR2')
```

Comment:
Since this is a "readme" (as opposite to API documentation), a descriptive (natural language) could be easier to the reader. I suggest dropping curly brackets.
```suggestion
  Initially supported options: 'GAIADR2' and 'GAIADR1'. (Default='GAIADR2')
```

Reply:
Addressed in e6ae123.